recipes/kosudoku/kosudoku-verify

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -e
```

Comment:
Why are those scripts included in this repo, if you can copy them from the github checkout?

Reply:
I wrote these 5 bash scripts, they are not part of the GH repo.

Reply:
Create a fork and add yours, that's maybe also good for fixing the old stuff in the future.

Reply:
Done